### ISSUE TITLE
Add --error to only output log messages of error or higher

### ISSUE THREAD

**Issue by tgharold** [borgbackup/borg#826]
Back in 2015, the following was added, but refactored out prior to the 1.0.0 release:

https://github.com/borgbackup/borg/commit/25140e8c8218fe19ef9ffc8029e3f53516b2d546#diff-dfaa777fd33eac4e20768ac811e52696R649

That gave the ability to specify the log level for all levels (debug, info, warning, error, critical).  The current default seems to be to only output "warning" or higher, but there is no way to tell borg that you only want to see "error" or higher.

Adding "--error" would allow me to suppress warning messages in the output.

**Comment by enkore** [comment]
There's the BORG_LOGGING_CONF environment variable, which uses these: https://docs.python.org/3/library/logging.config.html

**Comment by Ape** [comment]
Usually applications have `-q`, `--quiet` for this.

**Comment by enkore** [comment]
To suppress warnings?

**Comment by ThomasWaldmann** [comment]
quiet is the default, but it does not include warnings and errors.

and I am rather tending to remove -v / --verbose than to add -q / --quiet (because it seems to trigger expectations about the way it works that are not like it really works).

that said, adding --error might be an option if somebody prefers to not have some specific (and expected) warnings show up. @tgharold had the specific issue that a `borg check` cron job always emails him that "orphan chunks check skipped" warning because he only checked the last 5 archives, not all.

this could be fixed in 3 ways (to stop the spamming):
- add --error so one can decide to not see warnings (which might have bad side effects in case some important warning passes unnoticed)
- change log level of that message to info (orphan chunks are no big problem, the just waste space, but do not cause malfunction)
- kill that message completely

**Comment by enkore** [comment]
```
            logger.warning('Orphaned objects check skipped (needs all archives checked).')
```

This should be on INFO level imho.

**Comment by ThomasWaldmann** [comment]
@enkore agreed, that's a good reason.

**Comment by ThomasWaldmann** [comment]
While the one specific msg was fixed by changing the level, --error is not implemented yet (but will come in a minute).

**Comment by ThomasWaldmann** [comment]
fixed in 1.0-maint branch, soon to be merged into master also.
